src/App.js

```diff
@@ -89,14 +99,48 @@ function App() {
     setCardsText([...newCards]);
   };
 
+  const addToDeleteListHandler = (cardId, isSelected, isEdited) => {
```

Comment:
Actually this is more like 'Selection list' rather then specifically 'Deletion List'.